### PR TITLE
match as alternative to if and unless

### PR DESCRIPTION
**Pull request by cindRoberta**

### CODE REVIEW

File: doc/docs.md [hunk 1]
```diff
@@ -1572,6 +1572,28 @@ s := match number {
...
+
+or as an `unless` alternative: [unless Ruby](https://www.tutorialspoint.com/ruby/ruby_if_else.htm)
+
+```v
+match false {
+	2 > 4 { println('if') }
+	3 == 4 { println('else if') }
+	2 == 2 { println('else if2') }
+	else { println('else') }
+} // 'if' is printed
```
Comment (JalonSolov) [inline doc/docs.md:1594]: `else if2` should be printed.

**Comment by spytheman** [comment]
The examples in the documentation have to be formatted.
You can check locally with: `./v check-md doc/docs.md`, and you can use 3 backticks, followed by `v nofmt` if you have a reason to *not* format a specific block (formatting it, and just using `v` instead of `v nofmt`, is of course better).

File: doc/docs.md (hunk 1 above)
Comment (cindRoberta) [reply doc/docs.md]: done

**Comment by cindRoberta** [comment]
> The examples in the documentation have to be formatted.
> You can check locally with: `./v check-md doc/docs.md`, and you can use 3 backticks, followed by `v nofmt` if you have a reason to _not_ format a specific block (formatting it, and just using `v` instead of `v nofmt`, is of course better).

i don't want to download, ok I'll download

**Comment by cindRoberta** [comment]
@spytheman done

**Pull request merged**